components/EnoughToBuy.tsx

```diff
@@ -1,16 +1,46 @@
+import { motion, useAnimate } from "framer-motion";
+import { useState, useEffect } from "react";
+import { useAccount } from "wagmi";
+
 import { getImpactItems, isLiftPersonOutOfPovertyImpactItem } from "@/utils";
 
+import type { GetImpactItem } from "@/utils";
+
 type Props = {
   yearlyYield: number;
 };
 export default function EnoughToBuy({ yearlyYield }: Props) {
-  const yearlyImpactItem = getImpactItems(yearlyYield)[0];
+  const yearlyImpactItems = getImpactItems(yearlyYield);
   const enoughToLiftPersonOutOfPoverty =
-    yearlyImpactItem && isLiftPersonOutOfPovertyImpactItem(yearlyImpactItem);
+    yearlyImpactItems[0] &&
+    isLiftPersonOutOfPovertyImpactItem(yearlyImpactItems[0]);
+  const impactItemOffset = (yearlyImpactItems.length - 1) * -24;
+  const [scope, animate] = useAnimate();
+  const { isConnected } = useAccount();
+  useEffect(() => {
+    if (isConnected) {
+      const animation = async () => {
+        await animate("ul", { opacity: 1 }, { duration: 1 });
```

Comment:
This could be helpful for another time - [sequence](https://www.framer.com/motion/animate-function/##animate-sequences)